### ISSUE TITLE
Need setup how to

### ISSUE THREAD

**Issue by jjdeprisco** [plule/theremotion#43]
So far I have my Leap Motion controller set up on my Win 10 machine and can run Theremotion, but there is no sound. Installed Faust, but still not getting any sound. Do you have a better install checklist that shows specifically how all these elements communicate with one another?

Also, I was hoping Theremotion also generated MIDI for use in other apps, but apparently that is not part of the current feature set. Maybe something for the future? That way you would be able to use this with any VST or soft synth in Ableton, etc.

**Comment by plule** [comment]
@jjdeprisco It would make sense to make a controller out of it. I think it would be a bit hard to find appropriate synths, since we are not controlling with note on/note off, but volume pitch. Some people on reddit pitched in the direction of MPE synths, they can be controlled this way. Control voltage would also be a nice match. However, I have no real plan in that direction for now, the most fun I'm having with this project is by attempting to make a self contained instrument out of it that I can plug and play. I'm not using VSTs, nor I have an appropriate hardware synthesizer, which makes work in that direction unlikely, at least for now.

Sooo maybe some day but don't hold your breathe :)

**Comment by jjdeprisco** [comment]
Yeah, I really like the direction you are going to start with. But I think with some creative value scaling, you could make MIDI output happen without too much trouble. Your Theremotion app is obviously already outputting notes in some form, which can be mapped to MIDI notes, and the volume from one the hand could be translated into MIDI velocity. As for the on/off for the note, it seems to me you already have that in some form because the notes (when using a scale in Theremotion) appear fairly discreet. All depends on the end goal. I'm glad to see anything happening in this space as my Leap has been collecting dust for a while!

**Comment by plule** [comment]
Taking note of the OpenTheremin approach to this problem: https://www.gaudi.ch/OpenTheremin/index.php/opentheremin-v4/midi-interface

There is quite a significant step to go from how Theremotion operates today to a MIDI friendly one, but it's obviously doable. I'm quite convinced that the "right" way to make a controller out of it is CV to a modular system, but it could be due to my general ignorance of the VST world.

**Comment by jjdeprisco** [comment]
CV would be fine for many cases. I can see a possible interaction between Theremotion and programs like VCV Rack. Or maybe using MAX with Ableton, you could easily do something with CV to MIDI. For sure it's a niche thing. Even more niche than it already is!
